airflow-core/src/airflow/ui/src/pages/Events/Events.tsx

```diff
@@ -41,6 +42,43 @@ const eventsColumn = (
       skeletonWidth: 10,
     },
   },
+  {
+    accessorKey: "event",
+    enableSorting: true,
+    header: "Event",
+    meta: {
+      skeletonWidth: 10,
+    },
+  },
+  {
+    accessorKey: "owner",
+    enableSorting: true,
+    header: "User",
+    meta: {
+      skeletonWidth: 10,
+    },
+  },
+  {
+    accessorKey: "extra",
+    cell: ({ row: { original } }) => {
+      if (original.extra !== null) {
+        try {
+          const parsed = JSON.parse(original.extra) as Record<string, unknown>;
+
+          return <RenderedJsonField content={parsed} jsonProps={{ collapsed: true }} />;
```

Comment:
One one side I understand that per default you want to have it collapsed... on the other side if I am looking for a specific event it is also burdensome to expand all results individually? No real opinion but might also be hard to use if collapsed... but expanded per default is also not great... mhm...

Reply:
I was thinking of putting a button to toggle all of them on the page at once

Reply:
Maybe even the elements not being JSON could be "reduced text" to e.g. 50 chars only and then expand... but this would require a new widget I assume... but could be also a future PR... no pressure 8-)

Reply:
Updated. But yeah, I'll save a custom hideable text component for another PR